recipes/ocrmypdf/meta.yaml

```
@@ -0,0 +1,65 @@
+{% set name = "ocrmypdf" %}
+{% set version = "11.6.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ocrmypdf-{{ version }}.tar.gz
+  sha256: 0f624456a50be0b0bc8c0b59704d159f637616c093a1cabe8bb383706561bcf7
+
+build:
+  number: 0
+  skip: true   # [py2k or win]
```

Can this be made `noarch: python`? It's fine if the Windows CI then fails with missing dependencies
```suggestion
  noarch: python
```
You'll also have to remove the `# [not win]` selectors to make the linter happy

---

Yes, that is much better! (Too bad there isn't a way to do that without failing CI checks.) Done in https://github.com/conda-forge/staged-recipes/pull/14066/commits/4209d339b0b844bd1ddb5d7614ff5b16ccb19125

---

It only affects staged-recipes so it's not too bad. We could work around it in various ways but there's an endless list of more important things to work on 😄